ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/flow_tracking_settings.schema.yml

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  flow_tracking_settings:
+    documentation_options:
+      table: wan-preview
```

Comment:
```suggestion
```

Reply:
If you do this and rerun schema it will create a file with the table
```
roles/eos_designs/docs/tables/flow-tracking-settings.yml
```

Then you need to edit

```
roles/eos_designs/docs/wan-preview.md
```

To add the table, you can see how it is done for other sections